import-export-cli/integration/testdata/TestArtifactDirectory/DevFirstSampleCaseArtifacts/customAddLogMessage.j2

```diff
@@ -0,0 +1,3 @@
+<log level="custom">
+     <property name="{{propertyName}}" value="{{propertyValue}}"/>
+</log>
```

Comment:
shall we add new line?

Reply:
Fixed via https://github.com/wso2/product-apim-tooling/pull/891/commits/683e48b4a82057a7c628ceaa83c02282ea651f95